src/components/cards/CardMenu.vue

```diff
@@ -103,9 +107,9 @@ export default {
 			currentBoard: state => state.currentBoard,
 		}),
 
-		card() {
+		/* card() {
```

Comment:
:broom:

Reply:
There are also still usages of `this.id` in this file which is no longer available